subsys/net/l2/ethernet/gptp/gptp.c

```diff
@@ -629,6 +629,10 @@ void gptp_set_time_itv(struct gptp_uscaled_ns *interval,
 			/* Overflow, set maximum. */
 			interval->low = UINT64_MAX;
 			interval->high = UINT32_MAX;
+		} else if (log_msg_interval > 64) {
```

Comment:
Why not add "&&" condition to the "if" above?

Reply:
That can be done, will send a new version.